frontend/src/app/api/login/route.ts

```diff
@@ -0,0 +1,16 @@
+import { PrismaClient } from "@prisma/client";
+import { NextResponse } from "next/server";
+const prisma = new PrismaClient();
```

Comment:
should this be placed here? I feel like a line like this should be placed further up rather than inside a random login route. but idk.